gravitee-am-ui/src/app/domain/settings/password-policies/password-policies-idp-select-dialog/password-policies-idp-select-dialog.component.html

```diff
@@ -0,0 +1,72 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<div class="pp-container">
+  <h5>Add Identity Providers (IdP)</h5>
+  <p><small>Select Identity Providers to be associated to this policy.</small></p>
+  <mat-tab-group>
+    <mat-tab>
+      <ng-template mat-tab-label>
+        <span
+          class="tab-title"
+          matBadge="{{ result.idpsToLink.length }}"
+          matBadgeHidden="{{ result.idpsToLink.length === 0 }}"
+          matBadgeOverlap="false"
+          matBadgePosition="after"
+          >Unlinked</span
+        >
+      </ng-template>
+      <app-password-policies-idp-select-table
+        [showAssociation]="false"
+        [rows]="data.unlinkedIdps"
+        [withSelectionWarning]="false"
+        (selectionEvent)="handleEvent($event, this.result.idpsToLink)"
+      >
+      </app-password-policies-idp-select-table>
+    </mat-tab>
+    <mat-tab>
+      <ng-template mat-tab-label>
+        <span
+          class="tab-title"
+          matBadge="{{ result.idpsToUnlink.length }}"
+          matBadgeOverlap="false"
+          matBadgeHidden="{{ result.idpsToUnlink.length === 0 }}"
+          matBadgePosition="after"
+          >Linked to Existing Policy</span
+        >
+      </ng-template>
+      <div fxLayout="row" fxLayoutAlign="start center" class="warning-box">
+        <div fxFlex="initial" fxLayoutGap="5" fxLayoutAlign="start center"><mat-icon>info_outline</mat-icon></div>
+        <div fxFlex fxLayoutGap="5">
+          <p>An Identity Provider (IdP) can only by linked to one password policy.</p>
+          <p>Selecting a different IdP will unlink it form its current policy.</p>
+        </div>
+      </div>
+      <app-password-policies-idp-select-table
+        [showAssociation]="true"
+        [rows]="data.linkedIdps"
+        [withSelectionWarning]="true"
```

Comment:
**question**: do we need this input ? Do you think we can rely only on showAssociation as it look like the warning is only when we select an already associated idp?

Reply:
I have the impression that those properties are coincidentally similar. Substituting 'showAssociation' for 'withSelectionWarning' in 
`<mat-icon class="warning-icon" *ngIf="withSelectionWarning && selectionModel.get(row.id)">info_outline</mat-icon>`
might not make sense to a 'code vistior'. WDYT?